.github/workflows/e2e-qat.yml

```diff
@@ -0,0 +1,35 @@
+name: e2e-qat
+on:
+  workflow_dispatch:
+    inputs:
+      images:
+        description: 'Images to build before provisioning pull on worker'
+        required: true
+        default: 'intel-qat-plugin:devel crypto-perf:devel'
+
+env:
+  IMAGES: ${{ github.event.inputs.images }}
+
+jobs:
+  e2e-qat:
+    name: e2e-qat
+    #if: contains('["bart0sh"]', github.actor)
```

Comment:
@bart0sh some small leftover?

Reply:
I left it here in purpose, to show how to secure it even more. Can remove if it look ugly.

Reply:
It's OK like that.